src/amo/components/SectionLinks/index.js

```diff
@@ -2,40 +2,71 @@ import classNames from 'classnames';
 import React from 'react';
 import { compose } from 'redux';
 import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
 
 import Link from 'amo/components/Link';
 import type { ViewContextType } from 'amo/reducers/viewContext';
+import { setClientApp } from 'core/actions';
 import {
   ADDON_TYPE_EXTENSION,
   ADDON_TYPE_THEME,
+  CLIENT_APP_ANDROID,
+  CLIENT_APP_FIREFOX,
   VIEW_CONTEXT_EXPLORE,
   VIEW_CONTEXT_HOME,
+  VIEW_CONTEXT_LANGUAGE_TOOLS,
 } from 'core/constants';
+import type { ApiStateType } from 'core/reducers/api';
```

Comment:
I tend to put the `import type` at the end of the `import` block, but I don't know if it is the rule here. Also, if you use Flow without `// @flow` at the top of the file, it will not have any effect if I remember correctly.

Reply:
Oh, I guess that's a fine way to organise them. I tend to sort things by the filename we're importing from but separating out types wfm. 👍

Reply:
Oh, and yeah I constantly forget to add the flow annotation, my bad 😞